### PR TITLE
Record no-calls as ?

### PR DESCRIPTION
**Pull request by zhaofengli**
A more complete fix for #11. Now no-calls are recorded as `?` in SNP-JSON.

### CODE REVIEW

**Comment by contra** [comment]
How do you feel about this format? It seems like everyone else has rallied around D for deletion and - for no call, right? Should we be different from that?

**Comment by zhaofengli** [comment]
I think `-` is okay, matching the SNPedia notation. Changing the format means breaking existing code, too.

File: README.md [hunk 1]
```diff
@@ -102,6 +102,18 @@ Explanation:
...
+### Differences between formats
+
+SNP-JSON, SNPedia and vendor formats use different notations to indicate indels and no-calls.
+
+| Format        | Insertion            | Deletion | No Call | *Reference* |
+|---------------|----------------------|----------|---------|-------------|
+| 23AndMe       | `I`                  | `D`      | `-`     | https://customercare.23andme.com/hc/en-us/articles/212196888-What-does-not-determined-or-not-genotyped-mean- |
+| AncestryDNA   | `I`                  | `D`      | `0`     | https://www.ancestry.com/dna/en/legal/us/faq#raw-6 |
+| FamilyTreeDNA | `I`                  | `D`      | `-`     | https://www.familytreedna.com/learn/autosomal-ancestry/universal-dna-matching/read-family-finder-raw-data-file/ |
+| SNP-JSON      | `I`                  | `-`      | `?`     | |
```
Comment (contra) [inline README.md]: shouldnt this say snp-json uses the letters as well?

**Comment by contra** [comment]
Published as 1.0.4 - thanks

**Pull request merged**